package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "distributed_scheduler",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "SET NODE_ENV=dev && node lib/index.js",
```

Comment:
NODE_ENV typically has "development" as a value and not "dev". Plus, it should not be set in the start script, rather they need to be passed as environment variables from the shell